### ISSUE TITLE
Add import openapi spec to cli 

### ISSUE THREAD

**Issue by anttiviljami** [opticdev/optic#21]

**Comment by acunniffe** [comment]
Hey @anttiviljami -- I'm in the final stages of integrating these changes over on the [oas-export feature branch](https://github.com/opticdev/optic/tree/feature/oas-export). 

Right now the plan is to provide a new flag to api init:
`api init --import path/to/openapi.json`

This will immediately open the OAS file in Optic's API designer locally. It will also have the side effect of creating the `.api` folder in the working directory so changes you make in Optic can be persisted in the API repo. 

We're also adding our API exporter to the CLI as part of this issue.
`api generate oas --output oas.json`

We're using the `generate` command because we plan to have many kinds of output targets(e.g. js-sdk, typescript-types, java-classes etc)

The OAS exporter supports all the higher order and Generic types that are available to you in Optic. Here's what the [Pagination Example](https://design.useoptic.com/examples/pagination) looks like as OAS:

Notice how, `PaginatedList[T]` becomes `allOf[ PaginatedList, {items: array[T]]} ]`

```json
openapi: 3.0.1
info:
  title: Pagination Example
  version: '95'
paths:
  /owners:
    get:
      operationId: request_BhmawMJu8m
      responses:
        '200':
          content:
            application/json:
              schema:
                allOf:
                  - $ref: '#/components/schemas/PaginatedList'
                  - type: object
                    properties:
                      items:
                        type: array
                        items:
                          $ref: '#/components/schemas/Owner'
          description: ''
  /pets:
    get:
      operationId: request_fWhN94kA3S
      responses:
        '200':
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/PetList'
          description: ''
components:
  schemas:
    GUID:
      type: string
      description: An identifier represented as a "string"
    Owner:
      type: object
      properties:
        name:
          type: string
        pets:
          type: array
          items:
            $ref: '#/components/schemas/PetId'
        id:
          type: string
          description: An identifier represented as a "string"
    OwnerId:
      type: string
      description: A reference to an entity of Owner
    OwnerList:
      allOf:
        - $ref: '#/components/schemas/PaginatedList'
        - type: object
          properties:
            items:
              type: array
              items:
                $ref: '#/components/schemas/Owner'
    PaginatedList:
      type: object
      properties:
        items:
          type: array
          items: {}
        cursor:
          type: string
    PaginationCursor:
      type: string
    Pet:
      type: object
      properties:
        name:
          type: string
        id:
          type: string
          description: An identifier represented as a "string"
    PetId:
      type: string
      description: A reference to an entity of Pet
    PetList:
      allOf:
        - $ref: '#/components/schemas/PaginatedList'
        - type: object
          properties:
            items:
              type: array
              items:
                $ref: '#/components/schemas/Pet'

```

**Comment by acunniffe** [comment]
This is implemented in `npm install 5.0.0-alpha.2 -g`

Thanks for the great idea @anttiviljami. Closing the issue now 👍
